src/core/DataSourceBase.ts

```diff
@@ -72,20 +75,16 @@ export abstract class DataSourceBase<TQuery extends DataQuery, TOptions extends
    * @param body - The request payload as a key-value map.
    * @param options - Optional configuration for the request. This can include:
    *   - `showingErrorAlert` (boolean): If true, displays an error alert on request failure.
-   *   - Any other properties supported by {@link BackendSrvRequest}, such as headers, credentials, etc.
-   * @param useApiIngress - If true, uses API ingress bypassing the UI ingress for the request.
+   *   - `useApiIngress` (boolean): If true, uses API ingress bypassing the UI ingress for the request.
+   *   - Any other properties supported by {@link RequestOptions}, such as headers, credentials, etc.
```

Comment:
With the stricter typing of the options object and putting doc comments on the `RequestOptions` type itself I think we can avoid duplicating the options documentation in all these spots.